tests/tt_metal/tt_metal/unit_tests/basic/device.cpp

```diff
@@ -352,7 +352,6 @@ TEST_F(DeviceFixture, TestDeviceToHostMemChannelAssignment) {
 
     for (const auto& [mmio_dev_id, device_group] : mmio_device_to_device_group) {
         EXPECT_EQ(tt::Cluster::instance().get_num_host_channels(mmio_dev_id), device_group.size());
-        EXPECT_EQ(tt::Cluster::instance().get_assigned_channel_for_device(mmio_dev_id), 0);
```

Comment:
there was no real reason for mmio device to be given channel 0 hugepage but curious if this was changed?